### ISSUE TITLE
how to split the wilddeepfake？

### ISSUE THREAD

**Issue by ziminMIAO** [VISION-SJTU/RECCE#15]

**Comment by ziminMIAO** [comment]
How to divide the wilddeepfake dataset into a validation dataset? Can you elaborate on your division strategy?

**Comment by XJay18** [comment]
Hi,
Since the official release of the WildDeepfake dataset is only split into a train set and a test set, to compare our method with others fairly, we choose the test set as the validation set for both our method and our re-implemented ones.

**Comment by XJay18** [comment]
Close due to inactivation. Please feel free to reopen this issue if you still have related problems.

**Comment by jesslos** [comment]
> How to divide the wilddeepfake dataset into a validation dataset? Can you elaborate on your division strategy?

Hi,
I currently want to try to test the wilddeepfake dataset, but I can’t find it. I contacted the original author and there has been no reply. Could you please share your dataset?
